app/app.rb

```diff
@@ -0,0 +1,19 @@
+require_relative 'server'
+
+class App < Sinatra::Base
+  set :views, File.expand_path('../views', __FILE__)
+  set :public_folder, File.expand_path('../../public', __FILE__)
+  use Rack::MethodOverride
+  register Sinatra::Partial
+  register Sinatra::Flash
+  set :sessions, true
+  set :partial_template_engine, :erb
+  enable :partial_underscores
+
+  def current_user(id)
+    User.first(:id => id)
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)